scipy/stats/tests/test_resampling.py

```diff
@@ -563,8 +563,8 @@ def test_bootstrap_alternative(method):
 
 def test_jackknife_resample():
     shape = 3, 4, 5, 6
-    np.random.seed(0)
-    x = np.random.rand(*shape)
+    rng = np.random.default_rng(5274950392)
+    x = rng.standard_normal(shape)
```

Comment:
The existing code is a uniform number generator, not a normal distribution.

```suggestion
    x = rng.random(size=shape)
```